tcf_website/templates/reviews/new_review.html

```diff
@@ -31,5 +31,27 @@ <h1 class="h2 mt-2 mb-4">Write a review</h1>
   $(function () {
     $('[data-toggle="tooltip"]').tooltip()
   });
+
+  // Checks if required fields for review were filled
+  // If required fields are filled, timeout and hide the submit button so that the user can't spam click it
+  $(document).ready(function () {
+    $("#submitBtn").click(function () {
```

Comment:
Agreed, but I sometimes had to switch like with "document.getElementById("reviewForm").submit();" because it wouldn't work with $("#...") for some reason when I tried?